### PR TITLE
Added more test 

### PR DESCRIPTION
**Pull request by sameera2004**

### CODE REVIEW

**Comment by sameera2004** [comment]
@danielballan this will give correct label_array from segmentation_rings() I will add more tests to that function. 
Anyway name changes to pixel_to_radius, pixel_to_phi in core.py module, created import errors in several other modules in skxray. I fixed those however, there is new error in test_calibration.py

File: skxray/roi.py [hunk 1]
```diff
@@ -264,6 +264,8 @@ def segmented_rings(edges, segments, center, shape, offset_angle=0):
     """
     agrid = core.angle_grid(center, shape)
 
+    agrid[agrid < 0] = 2*np.pi + agrid[agrid < 0]
```
Comment (danielballan) [inline skxray/roi.py:267]: Good catch. This was indeed broken, but I think this solution is correct in every case. I will merge your PR onto this PR, add a test to demonstrate the problem I am seeing, and propose a fix in a new commit.

Comment (sameera2004) [inline skxray/roi.py:267]: @danielballan Sure

**Comment by danielballan** [comment]
I merged this manually because there were some conflicts. For next time, you shouldn't have merged my roi-edits branch into your more_tests branch; you should have checked out a fresh branch starting from the latest version of the roi-edits branch. But no worries -- I got your commits in!

**Pull request closed without merging**